spring-context/src/test/java/org/springframework/validation/DataBinderTests.java

```diff
@@ -82,6 +82,17 @@
  */
 class DataBinderTests {
 
+	Validator spouseValidator = Validator.of(TestBean.class)
+			.apply((tb, errors) -> {
```

Comment:
Seeing `apply()` in this "DSL" seems counterintuitive to me.

I'm wondering if we should introduce a dedicated functional interface for this purpose (with a meaningful type name and meaningful method name) instead of reusing `BiConsumer`.

Reply:
The `apply` comes from `Function`; not `BiConsumer` (where it is called `accept`).

Reply:
Indeed. I got the interfaces confused. In any case, the point I was making is the same.